mitodl/micromasters
# Make client_authorization_id field optional in ProctoredExamGrade

#### Pre-Flight checklist

  
#### What are the relevant tickets?
Fix #4936

#### What's this PR do?
Make client_authorization_id field optional in ProctoredExamGrade

#### How should this be manually tested?
Run migrations. In Django admin create a ProctoredExamGrade for your learner. `client_authorization_id` should not be required.
